src/demo/pages/Home.tsx

```diff
@@ -58,7 +58,10 @@ export default function Home() {
   }, []);
 
   async function onButtonPress() {
-    api.ExternalUser.loginWithTonomy({ callbackPath: "/callback" });
+    api.ExternalUser.loginWithTonomy({
+      callbackPath: "/callback",
+      dataRequest: { username: true },
```

Comment:
so currently it's requried to request username? what will be the case when user send this {username:false}